code/MkPairForSubMarineAndLandCable.py

```diff
@@ -0,0 +1,77 @@
+import sqlite3
+import math
+
+def get_landing_point_coord_from_database(db_file):
+    landing_point_coord = []
+    conn = sqlite3.connect(db_file)
+    cursor = conn.cursor()
+    cursor.execute('SELECT lp.latitude, lp.longitude, lp.city_name, lp.country FROM landing_points lp;')
+    datas = cursor.fetchall()
+    for data in datas:
+        landing_point_coord.append(data)
+
+    conn.close()
+    return set(landing_point_coord)
+
+def get_phys_nodes_coord_from_database(db_file):
+    phys_nodes_coord = []
+    conn = sqlite3.connect(db_file)
+    cursor = conn.cursor()
+    cursor.execute('SELECT pn.latitude, pn.longitude, pn.city, pn.country \
+                    FROM phys_nodes pn \
+                    WHERE EXISTS ( \
+                        SELECT 1  \
+                        FROM standard_paths  \
+                        WHERE pn.city = standard_paths.from_city OR pn.city = standard_paths.to_city \
+                    );')
+    datas = cursor.fetchall()
+    for data in datas:
+        phys_nodes_coord.append(data)
+    conn.close()
+    return set(phys_nodes_coord)
+
+def haversine(coord1, coord2):
```

Comment:
If this isn't different from the other haversine maybe just import it (or use the geopy library)